services/consensusalgo/leanhelix/service.go

```diff
@@ -103,8 +105,8 @@ func (s *service) buildBlocksEventLoop() {
 		}
 
 		// validate the current proposed block
-		if s.blocksForRounds[s.lastCommittedBlockHeight + 1] != nil {
-			err := s.leaderCollectVotesForBlock(s.blocksForRounds[s.lastCommittedBlockHeight + 1])
+		if s.blocksForRounds[s.lastCommittedBlockHeight+1] != nil {
```

Comment:
why are you removing the spaces around the plus sign?

Reply:
Not me, it's `go fmt`. I'm running it automatically after every save.